etc/evince.profile

```diff
@@ -14,6 +14,7 @@ include /etc/firejail/disable-devel.inc
 include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 
+include /etc/firejail/whitelist-common.inc
```

Comment:
Including this would completely prevent any files (PDFs) from being accessed.
Remove this and I'll merge

Reply:
I'll take another look at what I did here regarding whitelist-common.inc. Thanks for enlightening me on this. Have to take a break for a few hours, but I'll pick things up later today.